### DISCUSSION TITLE
About the singer profile picture acquisition problem

### DISCUSSION THREAD

**Discussion by autunn** [navidrome/navidrome#2943] [Ideas]
At present, it is known that the introduction of the singer, the album image is obtained from lastfm, and the singer's avatar is obtained from spotify, but because of the naming problem, many times, the singer can not get the avatar from spotify, want to ask the developer to adjust it to see if it can get the singer's avatar from spotify after failing. Get artist avatars from lastfm or other platforms

**Comment by autunn** [comment]
Or add a button to change the artist's name directly on the navidrome web interface, so you can get the spotify artist's avatar

**Reply by deluan** [reply]
You can add an `artist.jpg` image, as described here: https://www.navidrome.org/docs/usage/artwork/#artists

The ability to add an image from the web UI will come in a future release.
